.travis.yml

```diff
@@ -123,6 +123,27 @@ matrix:
         - CMAKE_ADD="-Dcoverage=ON"
         - TARGET=coverage_report
         - SKIP_TESTS=true
+    # test-free builds
+    - <<: *linux
+      if: commit_message !~ /travis_run_/ OR commit_message =~ /travis_run_linux/ OR commit_message =~ /travis_run_cov/
+      compiler: gcc-8
+      name: coverage, gcc-8
+      env:
+        - MATRIX_EVAL="CC=gcc-8 && CXX=g++-8"
+        - BUILD_TYPE=Debug
+        - CMAKE_ADD="-Dtests=OFF"
+        - TARGET=coverage_report
+        - SKIP_TESTS=true
+    - <<: *linux
+      if: commit_message !~ /travis_run_/ OR commit_message =~ /travis_run_linux/ OR commit_message =~ /travis_run_cov/
+      compiler: clang-8
```

Comment:
Do we really need a clang and a gcc build for the no tests situation? Travis takes forever to run, so I would rather be conservative about adding more CI jobs. I think the main thing to test here is that the includes are correct, and so I don't think clang and gcc are going to behave differently in that regard (I'm not a build expert by any means, so I could be wrong). 

However, I am wondering if we should have a no unity build for this. @ximinez what do you think?